contrib/woff2/woff2_wrapper.cc

```diff
@@ -0,0 +1,46 @@
+#include <woff2/decode.h>
+#include <woff2/encode.h>
+#include <woff2/output.h>
+
+#include <cinttypes>
+#include <cstddef>
+
+extern "C" bool WOFF2_ConvertWOFF2ToTTF(const uint8_t* data, size_t length,
+                                        uint8_t** result, size_t* result_length,
+                                        size_t max_size) {
+  if (result) *result = nullptr;
+  if (result_length) *result_length = 0;
+  if (!data || !length || !result || !result_length) return false;
+  size_t final_size = ::woff2::ComputeWOFF2FinalSize(data, length);
+  if (final_size > (max_size ? max_size : woff2::kDefaultMaxSize)) return false;
+  uint8_t* buffer = static_cast<uint8_t*>(std::malloc(final_size));
```

Comment:
Why not simply use an `std::vector<uint8_t>` (or even a plain `std::string`) and use its `data()` instead of manually managing this with malloc/free?